gunicorn.conf.py

```diff
@@ -0,0 +1,22 @@
+# pylint: disable=invalid-name
+import gunicorn
+
+# Tell gunicorn to run our app
+wsgi_app = "cms.wsgi:application"
+
+# Replace gunicorn's 'Server' HTTP header to avoid leaking info to attackers
+gunicorn.SERVER = ""
+
+# Restart gunicorn worker processes every 1200-1250 requests
+max_requests = 1200
+max_requests_jitter = 50
```

Comment:
We might need to revist whether this is a safe enough buffer at scale. It might be worth using env vars.